docs/modules/dev-services/nav.adoc

```diff
@@ -1,6 +1,7 @@
 .xref:index.adoc[Dev Services]
```

Comment:
We could go one step further here and create a subgroup for databases. Thoughts?

```adoc
.xref:index.adoc[Dev Services]
* Databases
** xref:mariadb.adoc[MariaDB]
** xref:mysql.adoc[MySQL]
** xref:postgresql.adoc[PostgreSQL]
** xref:redis.adoc[Redis]
* xref:lgtm.adoc[Grafana LGTM]
* xref:ollama.adoc[Ollama]
```

Reply:
Done